libvast/src/http_api.cpp

```diff
@@ -43,6 +47,29 @@ auto parse_endpoint_parameters(
             return caf::make_error(ec::invalid_argument, "not a boolean value");
           return data{result};
         },
+        [&string_value](const list_type&) -> caf::expected<data> {
```

Comment:
Please also check that the list type's value type is a string type, i.e.,

```
if (not caf::holds_alternative<string_type>(lt.value_type())) {
  return caf::make_error(...);
}
```

Reply:
Added in https://github.com/tenzir/vast/pull/3219/commits/b8c93d423f5e4910261438079f6aa0f1f9a0cb50